commcare_connect/utils/tables.py

```diff
@@ -0,0 +1,9 @@
+import django_tables2 as tables
+
+
+class BaseTailwindTable(tables.Table):
```

Comment:
nit: this applies to most of the code, and can be done at the end if easier, but I don't think we want everything named tailwind like it is now. Much like we don't call existing tables "BootstrapTable," I don't think it adds anything, because when this gets merged, it will be the only table type